Sources/BraintreeCard/CreditCardPOSTBody.swift

```diff
@@ -0,0 +1,139 @@
+// swiftlint:disable nesting
+
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+// swiftlint:disable nesting
+struct CreditCardPOSTBody: Encodable {
+
+    var authenticationInsight: Bool?
+    var merchantAccountID: String?
+    var meta: Meta?
+    let creditCard: CreditCard?
+    
+    enum CodingKeys: String, CodingKey {
+        case authenticationInsight
+        case meta = "_meta"
+        case merchantAccountID = "merchantAccountId"
+        case creditCard = "credit_card"
+    }
+
+    init(
+        card: BTCard,
+        metadata: BTClientMetadata
+    ) {
+        self.creditCard = CreditCard(card: card)
+        
+        if card.authenticationInsightRequested {
+            self.authenticationInsight = card.authenticationInsightRequested
+            self.merchantAccountID = card.merchantAccountID
+        }
+        
+        self.meta = Meta(metadata: metadata)
+    }
+
+    struct Meta: Encodable {
+
+        var integration: String
+        var source: String
+        var sessionID: String
+
+        init(metadata: BTClientMetadata) {
+            self.integration = metadata.integration.stringValue
+            self.source = metadata.source.stringValue
+            self.sessionID = metadata.sessionID
+        }
+
+        enum CodingKeys: String, CodingKey {
+            case integration
+            case source
+            case sessionID = "sessionId"
+        }
+    }
+
+    /// POST Body Model
```

Comment:
Take it or leave it but not sure this comment is helpful. Can likely remove once we add the one to the top of this struct.